src/components/ZoomableMap.tsx

```
@@ -145,3 +161,40 @@ function getHiddenStyles(zoomLevel: number): string {
 
   return hiddenStyles;
 }
+
+function getMapItemVisilibityStyle(
+  itemType: "planet" | "spacelane",
+  visibility: MapItemVisibility,
+): string {
+  switch (visibility) {
+    case "show":
+      return itemType === "planet"
+        ? styles.show_planets
+        : styles.show_spacelanes;
+    case "hide":
+      return itemType === "planet"
+        ? styles.hide_planets
+        : styles.hide_spacelanes;
+    case "dynamic":
+    default:
+      return "";
+  }
+}
+
+function getMapLabelVisibilityStyle(
+  itemType: "planet_label" | "spacelane_label",
+  visibility: MapItemVisibility,
+  itemVisibility: MapItemVisibility,
+): string {
+  if (itemVisibility === "hide" || visibility === "hide") {
```

issue: this will probably be confusing to end users. Instead of secretly changing the visibility of labels, can we change the actual value in the dropdown (and the available options) when the planet visibility is changed? There's some amount of nuance, see table below for proposed behavior:

|               | Planet Show              | Planet Dynamic                      | Planet Hide                                             |
| ------------- | ------------------------ | ----------------------------------- | ------------------------------------------------------- |
| Label Show    | Show labels              | Show labels for all visible planets | Set labels to hide and disable show and dynamic options |
| Label Dynamic | Default dynamic behavior | Default dynamic behavior            | Set labels to hide and disable show and dynamic options |
| Label Hide    | Hide labels              | Hide labels                         | Hide labels                                             |

---

An added benefit is that this will simplify the css slightly. Instead of using css to hide labels when planets are hidden, the labels class will change the visibility on it's own